File: c++_backup/Amlyan/OOP/pattern.cpp
#include<iostream>
using namespace std;
int main()
{
	int i,n,j;
	int a[20][20];
	cout<<"\n Enter the number:\n"<<endl;
	cin>>n;
	for(i=0;i<n;i++)
	{
		for(j=0;j<n;j++)
		{
			
			if(j==0||i==0)
			{		
				a[i][j]=i+j+1;	
			}	
			else if(i==n-1||j==n-1)
			{
				a[i][j]=2*n-i-j-1;
			}
			else
			{
				a[i][j]=0;
			}	
			
		}
	}
	for(i=0;i<n;i++)
	{
		for(j=0;j<n;j++)
		{
			if(a[i][j]!=0)
			{
				cout<<a[i][j]<<"\t";		
			}
			else
			{
				cout<<"\t";
			}
		}
		cout<<endl;
	}
	return 0;
}

File: 5th_sem_C_Java_VB/JAVA-SXC/AMLYAN1/pattern.java
import java.util.*;
class pattern
{
	public static void main(String args[])
	{
	Scanner sc=new Scanner(System.in);
	System.out.println("Enter a number n");
	int n=sc.nextInt();
	System.out.println("Enter the values in the matrix");
	int arr[][]=new int [n][n];
	for(int i=0;i<n;i++)
		for(int j=0;j<n;j++)
				arr[i][j]=sc.nextInt();
	int a=0;
	int b=1;
	for(int i=0;i<n;i++)
	{ a=i+1; 
	  b=0;
	for(int j=0;j<n;j++)
	{
		if(i<j)
		{
			int temp=arr[i][j];
			arr[i][j]=arr[a][b];
			arr[a][b]=temp;
			a=a+1;
			b=b+1;
		}
	}}
	for(int i=0;i<n;i++)
	{
		System.out.println();
		for(int j=0;j<n;j++)
			System.out.print(arr[i][j]+" ");
	}
}
}


